### ISSUE TITLE
Move all repos to GitHub action based CI

### ISSUE THREAD

**Issue by JBAhire** [hypertrace/hypertrace#144]
## Use Case
We were facing issues with CircleCI similar to https://github.com/hypertrace/hypertrace/issues/132 and also you need authorized user commit to get workflow running in the first place. We did analysis of GHA vs CircleCI and GHA seemed to be better overall option for Hypertrace CI use-case. 

We have already done POC on attribute service repo: https://github.com/hypertrace/attribute-service and we should move rest of the repos to use GitHub actions as well. 

## Proposal
We have created template workflows which you can import for each repo and modify according to requirements (similar to this: https://github.com/hypertrace/attribute-service/actions/new). 

This issue will track migration to GitHub Actions:
- [x] https://github.com/hypertrace/hypertrace-ui
- [x] https://github.com/hypertrace/hypertrace-collector [NA]
- [x] https://github.com/hypertrace/entity-service
- [x] https://github.com/hypertrace/hypertrace-core-graphql
- [x] https://github.com/hypertrace/hypertrace-graphql
- [x] https://github.com/hypertrace/document-store 
- [x] https://github.com/hypertrace/service-framework 
- [x] https://github.com/hypertrace/kafka-streams-framework 
- [x] https://github.com/hypertrace/attribute-service
- [x] https://github.com/hypertrace/hypertrace-ingester 
- [x] https://github.com/hypertrace/hyperdash-angular
- [x] https://github.com/hypertrace/config-bootstrapper
- [x] https://github.com/hypertrace/gateway-service
- [x] https://github.com/hypertrace/helm-gcs-packager {archiving} [NA]
- [x] https://github.com/hypertrace/hypertrace-service 
- [x] https://github.com/hypertrace/config-service 
- [x] https://github.com/hypertrace/kafka 
- [x] https://github.com/hypertrace/hyperdash
- [x] https://github.com/hypertrace/query-service
- [x] https://github.com/hypertrace/pinot 
- [x] https://github.com/hypertrace/zookeeper 
- [x] https://github.com/hypertrace/mongodb 
- [x] https://github.com/hypertrace/kafka-topic-creator 
- [x] https://github.com/hypertrace/schema-registry 
- [x] https://github.com/hypertrace/hypertrace-gradle-avro-plugin
- [x] https://github.com/hypertrace/hypertrace-gradle-integration-test-plugin
- [x] https://github.com/hypertrace/hypertrace-gradle-jacoco-report-plugin
- [x] https://github.com/hypertrace/hypertrace-gradle-publish-plugin 
- [x] https://github.com/hypertrace/hypertrace-gradle-repository-plugin 
- [x] https://github.com/hypertrace/hypertrace-gradle-docker-plugins 
- [x] https://github.com/hypertrace/hypertrace-gradle-version-settings-plugin
- [x] https://github.com/hypertrace/hypertrace-gradle-ci-utils-plugin 
- [x] https://github.com/hypertrace/data-model 
- [x] https://github.com/hypertrace/view-generator-framework 
- [x] https://github.com/hypertrace/hypertrace-core-view-generator 
- [x] https://github.com/hypertrace/event-store 
- [x] https://github.com/hypertrace/java-grpc-utils 
- [x] https://github.com/hypertrace/docker-java-base 
- [ ] https://github.com/hypertrace/javaagent [PR] { @pavolloffay will be taking care of changes on top of my PR }
- [x] https://github.com/hypertrace/goagent [PR]
- [x] https://github.com/hypertrace/opencensus-service {deprecating soon} [NA]

**Comment by JBAhire** [comment]
closing this as we are done with this effort. @pavolloffay will merge javaagent PR once he is done making changes with it.

**Comment by jcchavezs** [comment]
Awesome work @jbahire

**Comment by rish691** [comment]
This is so methodically done. Great job @JBAhire
